sdk/src/cli/context/common.rs

```diff
@@ -0,0 +1,254 @@
+use anyhow::{Context, Result};
```

Comment:
Yes I struggled to decide where to put this as well. It's not clear where it would be most logical. I'm happy to move this to cli/? (as long as we're ok to move the commands there as well, otherwise we would have a circular dependency).

Reply:
As discussed will leave this as is for now